brambling/templates/brambling/event/order/attendees.html

```diff
@@ -57,17 +55,18 @@
 										</a>
 									</div>
 								</div>
-								{% with items=attendee.bought_items.all %}
-									{% if items %}
-										<ul class='list-group'>
-											{% for item in items %}
-												{% if item.status == item.RESERVED or item.status == item.UNPAID or item.status == item.BOUGHT %}
-													<li class='list-group-item'>{{ item.item_option_name }} ({{ item.item_name }})</li>
-												{% endif %}
-											{% endfor %}
-										</ul>
-									{% endif %}
-								{% endwith %}
+								{% if attendee.items or attendee.errors %}
+									<ul class='list-group'>
+										{% for item in attendee.items %}
+											<li class='list-group-item'>{{ item.item_option_name }} ({{ item.item_name }})</li>
+										{% endfor %}
+										{% for error in attendee.errors %}
+											<li class='list-group-item list-group-item-danger'>
+												{{ error }}
+											</li>
+										{% endfor %}
+									</ul>
+								{% endif %}
```

Comment:
Um, not having any items, not having housing data filled out, not having basic data filled out.

https://github.com/littleweaver/django-brambling/pull/651/files#diff-8207526999f76beabfe2144669934c56R132